easybuild/easyconfigs/t/Trim_Galore/Trim_Galore-0.3.7-goolf-1.4.10.eb

```diff
@@ -16,7 +16,7 @@ source_urls = ['http://www.bioinformatics.babraham.ac.uk/projects/%(namelower)s'
 install_cmd = "cp -a %(namelower)s_zip/* %(installdir)s && chmod +x %(installdir)s/%(namelower)s"
 
 dependencies = [
-     ('FastQC', '0.10.1'),
+     ('FastQC', '0.10.1', '', True),
```

Comment:
sorry, the `''` should be `-Java-1.7.0_80`, i.e. the versionsuffix you used in the replacement FastQC easyconfig